docs/team.md

```diff
@@ -0,0 +1,6 @@
+# Team
```

Comment:
let's remove this, don't think we need to show the team in the docs :D also you forgot yourself :P

Reply:
I mention only the team leaders...
Maybe not a whole page (I don't have too much experience with open source...) but should we mention some people, a section, "who you can ask if you have questions..."...

Reply:
Uhm, I feel like questions should be asked as issues, so anyone could reply ;) and we also have a record of the questions  :)